tests_metricflow/snapshots/test_metric_time_dimension_to_sql.py/SqlQueryPlan/DuckDB/test_simple_query_with_metric_time_dimension__plan0_optimized.sql

```diff
@@ -5,40 +5,45 @@ docstring:
 sql_engine: DuckDB
 ---
 -- Combine Aggregated Outputs
+WITH rss_28020_cte AS (
+  -- Read Elements From Semantic Model 'bookings_source'
+  SELECT
+    1 AS bookings
+    , booking_value AS booking_payments
+    , DATE_TRUNC('day', ds) AS ds__day
+    , DATE_TRUNC('day', paid_at) AS paid_at__day
+  FROM ***************************.fct_bookings bookings_source_src_28000
+)
+
 SELECT
   COALESCE(subq_14.metric_time__day, subq_19.metric_time__day) AS metric_time__day
   , MAX(subq_14.bookings) AS bookings
   , MAX(subq_19.booking_payments) AS booking_payments
 FROM (
+  -- Read From CTE For node_id=rss_28020
+  -- Metric Time Dimension 'ds'
+  -- Pass Only Elements: ['bookings', 'metric_time__day']
   -- Aggregate Measures
   -- Compute Metrics via Expressions
   SELECT
-    metric_time__day
+    ds__day AS metric_time__day
     , SUM(bookings) AS bookings
-  FROM (
-    -- Read Elements From Semantic Model 'bookings_source'
-    -- Metric Time Dimension 'ds'
-    -- Pass Only Elements: ['bookings', 'metric_time__day']
-    SELECT
-      DATE_TRUNC('day', ds) AS metric_time__day
-      , 1 AS bookings
-    FROM ***************************.fct_bookings bookings_source_src_28000
-  ) subq_12
+  FROM rss_28020_cte rss_28020_cte
```

Comment:
Noticing this alias is unnecessary duplication. Same with line 44 below. Is that intended? Maybe it's required for certain engines?

Reply:
We always specify an alias for the sources in a `SELECT`. This is something that can be updated, but we haven't gotten around to it. We also always specify an alias for each of the columns, which is something that can be cleaned up as well.